.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -50,9 +50,25 @@ body:
         - 6.0.419
         - 6.0.424
         - 6.0.486
-        - 7.0 (current)
-        - 8.0 previews
-        - Unknown/Other (please specify)
+        - 6.0.536
```

Comment:
All of 6.x is no longer supported, so if someone is having an issue with that, it won't get fixed anyway. I suggest removing all 6.x releases.

Reply:
(Though it's probably fine to keep a _single_ 6.x release in the "originally worked in version" in case someone is reporting a regression from 6.x to current (and at that point it doesn't matter which 6.x it was).

Reply:
Thank you for the feedback